subcommands/config/log.go

```diff
@@ -25,10 +27,16 @@ func doConfigLog(cmd *cobra.Command, args []string) {
 	listLimit, _ := cmd.Flags().GetInt("limit")
 	group, _ := cmd.Flags().GetString("group")
 
+	lookups, err := subcommands.GetUsersLookupById(api, factory)
+	if err != nil {
+		fmt.Println("ERROR: Unable to look up factory user names", err)
+	}
```

Comment:
```suggestion
	subcommands.DieNotNil(err, "Failed to look up factory user names")
```

Plus, I think it would work well inside the `GetUsersLookupById`, as it was before.

Reply:
The struct didn't have a member for the api client. I wasn't sure a handle to that in an "Opts" struct made sense. I'm indifferent though. We could do a "lookup table callback func" but I thought that was getting out of hand.